src/quo2/components/avatars/channel_avatar/view.cljs

```diff
@@ -0,0 +1,58 @@
+(ns quo2.components.avatars.channel-avatar.view
+  (:require [clojure.string :as string]
+            [quo2.components.avatars.channel-avatar.style :as style]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            utils.string))
+
+(defn- initials
+  [full-name big? color]
+  (let [amount-initials (if big? 2 1)]
+    [text/text
+     {:accessibility-label :initials
+      :size                :paragraph-2
+      :weight              :semi-bold
+      :style               {:color color}}
+     (utils.string/get-initials full-name amount-initials)]))
+
+(defn- lock
+  [locked? big?]
+  ;; When `locked?` is nil, we must not display the unlocked icon.
+  (when (boolean? locked?)
+    [rn/view
+     {:accessibility-label :lock
+      :style               (style/lock-container big?)}
+     [icons/icon (if locked? :i/locked :i/unlocked)
+      {:color           (colors/theme-colors colors/neutral-50 colors/neutral-40)
+       :container-style style/lock-icon
+       :size            12}]]))
+
+(defn view
+  "Options:
+
+  :big? - bool (default nil) - Container size
```

Comment:
hmm, should we use actual sizes instead?

Reply:
@J-Son89, if you ask me this question my first instinct is to say yes, because "magic sizes" being passed as arguments are a leaking abstraction OMO. If another designer decides to change a single pixel, many calling sites will need to change. I think the code should be more resilient to arbitrary changes in the design system.

I prefer semantic names, but at the same time, I don't like that each component uses different names to mean different sizes. But man I dislike those 32/20/etc numbers flying around as arguments in the repo...

Reply:
Yep, I can agree to that. We can leave for now but I think we should figure out a consistent approach some point soon.

My point here was rather than 'big',
having :size with values :big or :small 
But component can be adjust easily if more sizes are added

Reply:
Good suggestion for changing from `big?` to `:size` 👍🏼 I'll apply it now.

> Yep, I can agree to that. We can leave for now but I think we should figure out a consistent approach some point soon.

💯